org.eclipse.pde.doc.user/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.pde.doc.user; singleton:=true
-Bundle-Version: 3.15.300.qualifier
+Bundle-Version: 3.15.400.qualifier
```

Comment:
Btw. does anybody know the policy when to bump the minor and when to bump the micro/service version of a doc bundle? Because it is usually not used from code (isn't it?) that bundle itself does not really have an API and versioning according to that not really possible, I think?

Reply:
At one point in the history is was 3.14.2200.qualifier suggesting 22 +100 increments.  I doubt there is ever a compelling reason to increment the minor version.

Reply:
That's right. Just incrementing the service version forever does not look suitable.
We could just increment the minor version when new API is added or some is removed after the deprecation period. But this could mean that the minor version also raises relatively quickly since the doc is some kind of an aggregator. Again on the other hand it still wouldn't be higher than the aggregator features since they are incremented every time.

Reply:
One could think about just increment the minor version on each release and therefore reset the micro version to zero.

Reply:
That would be possible, but for this change I think it is sufficient if we agree to bump the minor version.

Reply:
What if we align the version of PDE Doc bundle with the version of `org.eclipse.pde` feature? The argument could be: PDE Doc bundle content should be in sync with a content of main PDE feature.

Reply:
Fair point.
Although this means that I had to bump the minor version of the o.e.pde feature.
I assumed the p2 version checker would suggest to bump a feature's minor version if a plugin's minor has changed. 🤔
But maybe I confused this.